news/2020-09-05-mappers-report-august.md

```diff
@@ -0,0 +1,195 @@
+- **[Add a more specific rule for usage of .ogg files as beatmap audio](https://osu.ppy.sh/community/forums/topics/1123516)**
+
+### osu!
+
+- **[Add a rule to limit slider curve nodes](https://osu.ppy.sh/community/forums/topics/1121195)**
+
+### osu!taiko
+
+- **[Regulate Beginner difficulties](https://osu.ppy.sh/community/forums/topics/1125005)**
+
+### osu!mania
+
+- **[Change some duplicated phrases in 7K hard guidelines](https://osu.ppy.sh/community/forums/topics/1112674)**
+- **[Rule regarding omitting first bar line](https://osu.ppy.sh/community/forums/topics/1122014)**
+
+## <a id="within-bn-nat"></a>Within the Beatmap Nominators and NAT
+
+### BN Additions
+
+- **osu!:** [Sylas](https://osu.ppy.sh/users/3906405), [Ryuusei Aika](https://osu.ppy.sh/users/7777875), [NeKroMan4ik](https://osu.ppy.sh/users/11387664), [Karen](https://osu.ppy.sh/users/3143784), [Een Persoon](https://osu.ppy.sh/users/10959501), [Pentori](https://osu.ppy.sh/users/7452237), [Firika](https://osu.ppy.sh/users/9590557) & [Mimari](https://osu.ppy.sh/users/14339830)
+- **osu!taiko:** [Ulqui](https://osu.ppy.sh/users/1263669) & [Jaltzu](https://osu.ppy.sh/users/2597417)
+- **osu!catch:** [Rocma](https://osu.ppy.sh/users/566276), [JierYagtama](https://osu.ppy.sh/users/7483452) & [autofanboy](https://osu.ppy.sh/users/636114)
+- **osu!mania:** [Murumoo](https://osu.ppy.sh/users/8001433), [RiP46](https://osu.ppy.sh/users/5222596) & [Stelar](https://osu.ppy.sh/users/15205070)
+
+Do you want to see your own name on this list? Do you like modding and helping other people with their beatmaps? If your answer to both of these questions is "yes", then you can also become a Beatmap Nominator! The applications are always open. 
+
+Head over to the [Beatmap Nominator website](https://bn.mappersguild.com) and apply today!
+
+### BN Departures
+
+- **osu!taiko:** [Yales](https://osu.ppy.sh/users/2377881) & [Axer](https://osu.ppy.sh/users/7299864)
+- **osu!catch:** [Dako](https://osu.ppy.sh/users/11081858)
+- **osu!mania:** [Kyousukee](https://osu.ppy.sh/users/8842107)
+
+## <a id="mappers-showcase"></a>Mappers' Showcase
+
+It's time again for the Mappers' Showcase! As every month we would like to present you some lesser-known mappers which are doing great content! 
+As we finished the first cycle with all modes and the Storyboarder showcase, we're going back to the beginning, starting again with the osu! game mode.
+
+Thanks to [Pachiru](https://osu.ppy.sh/users/2850983) and [hypercyte](https://osu.ppy.sh/users/9155377) for the selections of this month.
+
+### Scyhei
+
+Hidden in the shadow, [Scyhei](https://osu.ppy.sh/users/10772337) have been in the mapping scene for a few years already and have been constantly improving and innovating their mapping over the years. Despite having a lot of unfinished or unpublished sets, their mapping mastery in the wide panel of song-style and mapping style is what gives them a lot of potential to stand out in the mapping scene. 
```

Comment:
- Scyhei HAS been in the mapping scene, not have
- and HAS been constantly improving, not have
- change "over the years" to "over this time" to stop the duplication of years in the earlier part of the sentence
- I can't figure out what you mean by the "wide panel" of styles, do you mean like wide varience?